definitions/ext-rf_scanner/summary_metrics.yml

```
@@ -0,0 +1,48 @@
+msToFirstRead:
+  title: Request Time (ms)
+  
+  query:
+    select: average(msToFirstRead)
+    from: THD_RF_TRANSACTION
+    facet: macAddress
+    
+msInRead:
+  title: Response Time (ms)
+  
+  query:
+    select: average(msInRead)
+    from: THD_RF_TRANSACTION
+    facet: macAddress
+
+bytesRead:
+  title: Bytes Read
+  
+  query:
+    select: average(bytesRead)
+    from: THD_RF_TRANSACTION
+    facet: macAddress
+
+bytesWritten:
+  title: Bytes Written  
+  
+  query:
+    select: average(bytesWritten)
+    from: THD_RF_TRANSACTION
+    facet: macAddress
+
+totalTime:
+  title: Total Time  
+  
+  query:
+    select: average(totalTime)
+    from: THD_RF_TRANSACTION
+    facet: macAddress
+    
+actions:
+  title: Actions 
+  
+  query:
+    select: count(action)
+    from: THD_RF_CONNECTION_EVENT
+    facet: macAddress, action
```

```suggestion
    eventId: entity.guid
```

We don't support multiple facets right now